_scripts/import_reference_docs.js

```diff
@@ -152,28 +152,44 @@ ghrepo.contents('extensions', "master", function(err, data) {
 
 		var order = index;
 
+		// use the last part of the path as the component name
+		var nameIndex = component.path.lastIndexOf('/') + 1;
+		var componentName = component.path.slice(nameIndex);
+
 		ghrepo.contents(component.path, "master", function(err, data) {
 
-			// fish out the markdown file from the folder
-			var component;
+			// fish out the README file from the folder
+			var componentReadme;
 			for (var i = 0; i < data.length; i++) {
+				// Find the first `.md` file in the component path data. If there is a
+				// `README.md`, select that as the component README instead.
+				// NOTE: The selection of the first `.md` file is done to ensure
+				//   backwards compatibility with an older file structure in the
+				//   'extensions' directory. Rather than a `README.md`, each extension
+				//   had a `.md` file with the extension name (e.g. 'amp-font.md') which
+				//   was meant to be used as the README for that extension. When all
+				//   extension READMEs have been changed to `README.md`, this loop can
+				//   be simplified.
 				if(data[i].type === 'file' && /\.md$/.test(data[i].name)) {
-					component = data[i];
-					break;
+					if (!componentReadme) {
```

Comment:
Capturing the first `.md` in the component matches the previous import convention. The new naming convention for primary readmes of extension components is `README.md` (e.g. `amp-font/amp-font.md` -> `amp-font/README.md`).